arch/emscripten/web/src/index.js

```diff
@@ -61,6 +61,29 @@ class LoadingScreen {
     }
 }
 
+window.MzxrunInitShortcuts = function() {
```

Comment:
I think this should be moved into an inner method - I prefer only exposing the MzxrunInitialize interface, and all other calls (if any) to be done via a callback-returned object.